tools/mininet/BUILD

```diff
@@ -0,0 +1,32 @@
+#
+# Copyright 2020-present Open Networking Foundation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+load("@rules_pkg//:pkg.bzl", "pkg_tar", "pkg_deb")
+
+licenses(["notice"])  # Apache v2
+
+package(
+    default_visibility = ["//visibility:public"],
+)
+
+pkg_tar(
```

Comment:
we might consider a wheel rather than a tarball here

Reply:
The `py_wheel` rules are experimental and not available in the released version yet. I also did not see a way to structure the files like we need them.